@ecomplus/storefront-app/src/components/js/OrdersListPagination.js

```diff
@@ -0,0 +1,90 @@
+import {
+  i19previous,
+  i19next
+} from '@ecomplus/i18n'
+
+import { i18n } from '@ecomplus/utils'
+
+export default {
+  props: {
+    pageSize: {
+      type: Number,
+      default: 10
+    },
+    pageLimit: {
+      type: Number,
+      default: 5
+    },
+    itemsLength: {
+      type: Number,
+      default: 0
+    }
+  },
+  data () {
+    return {
+      page: 1
+    }
+  },
+
+  computed: {
+    i19next: () => i18n(i19next),
+    i19previous: () => i18n(i19previous),
+
+    pageQuantity () {
+      return Math.ceil(this.itemsLength / this.pageSize) || 0
+    },
+
+    currentPage: {
+      get: function () {
+        return this.page
+      },
+      set: function (page) {
+        const from = (page - 1) * 10
+        this.page = page
+
+        this.$emit('update-orders', from)
+      }
+    },
+
+    pages () {
+      const pages = []
+
+      for (let i = 1; i <= this.pageQuantity; i++) {
+        pages.push(i)
+      }
+
+      return pages
+    },
```

Comment:
Pra quê esse array sendo que ele sempre vai ser só uma contagem até o `this.pageQuantity`? 🤔